app/utils/NextPage.scala

```diff
@@ -71,6 +71,7 @@ object NextPage
     with HaveYouRegisteredTrustNextPage
     with DoYouHaveCHIEFRoleNextPage
     with DoYouWantToAddImportExportNextPage
+    with utils.nextpage.vat.moss.WhereIsYourBusinessBasedNextPage
```

Comment:
add import utils.nextpage.vat.moss.WhereIsYourBusinessBasedNextPage to shorten this line